### DISCUSSION TITLE
How can I copy a single cell?

### DISCUSSION THREAD

**Discussion by needACoffee** [Sequel-Ace/Sequel-Ace#1723] [Q&A] [answered]
Maybe I missed something, but when I am on a field, I can only copy the complete row. How can I copy the single field I am over? That would be very helpful.

**Comment by samuelbradshaw** [comment, marked as answer]
When you double-click on a cell, that cell becomes active for editing and the contents of the cell are selected. From there, you can copy-paste.

**Comment by needACoffee** [comment]
Of course, your right. But I don't like it to get in the possibility to accidently change something - just to copy it. I think, an additional option for the right mouse click context menu like 'Copy cell' would be a good idea. But generally it's sufficient of course.
